ScanToShop/ViewControllers/RegisterViewController.m

```diff
@@ -7,8 +7,12 @@
 //
 
 #import "RegisterViewController.h"
+#import "AlertManager.h"
+#import "DatabaseManager.h"
+#import <Parse/Parse.h>
```

Comment:
How can you not import parse here? Parse is an implementation detail of your database and should be hidden from your app imo.